### PR TITLE
Frontend/Compact settings page

### LINKED ISSUE #319: Compact Admins add authorize.net account info (frontend)
- see #280

### PR DESCRIPTION
**Pull request by jsandoval81**
### Requirements List
- `yarn install --ignore-engines`

### Description List
- Add a CompactSettings page & route
- Add Compact Settings to the main nav for compact admins
- Add a PaymentProcessorConfig UI component
- Update the Compact model with a new interface type for the payment processor config
- Update the data API with the compact payment processor config endpoint call
- Allow the password input to optionally hide the password requirements
- Update the vue-i18 dependency minor version to address blocking build warning
    - https://github.com/csg-org/CompactConnect/security/dependabot/59
    - https://github.com/csg-org/CompactConnect/security/dependabot/60
    - https://github.com/csg-org/CompactConnect/security/dependabot/61
    - https://github.com/csg-org/CompactConnect/security/dependabot/62
- Add type shim to support re-working of types in the new vue-i18n version
- Updated the automated `yarn audit` command to actually honor excluded severity levels

### Testing List
- `yarn test:unit:all` should run without errors or warnings
- `yarn serve` should run without errors or warnings
- `yarn build` should run without errors or warnings
- Code review
- Should not see "Compact Settings" in the main nav if not logged in as a compact admin
- Should see "Compact Settings" in the main nav if logged in as a compact admin
    - Navigate to Compact Settings page
    - All values should be required
    - Values that the payment server can't verify will return an expected server error
    - For the IA Test environment, you can get the valid values from @jsandoval81 in slack

Closes #319

### CODE REVIEW

**Comment by ChiefStief** [comment]
Im still getting one low vuln thats blocking serving after re-installing `node_modules`

**Comment by jsandoval81** [comment]
@ChiefStief It looks like the `--levels` option of the `yarn audit` command is display only and will still return a non-zero exit code to stdout for cases that are trying to be ignored.

I wrapped the command to also evaluate the exit code and handle according to our desired severity level. Yarn docs on their exit codes are [here](https://classic.yarnpkg.com/lang/en/docs/cli/audit/#toc-yarn-audit).

File: webroot/src/components/PaymentProcessorConfig/PaymentProcessorConfig.vue [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                    <InputPassword
+                        :formInput="formData.accountKey"
+                        :joiMessages="joiMessages"
+                        :showEyeIcon="true"
+                        :showRequirements="false"
+                    />
+                    <InputSubmit
+                        :formInput="formData.submit"
+                        :label="submitLabel"
+                        :isEnabled="!isFormLoading"
```
Comment (ChiefStief) [inline webroot/src/components/PaymentProcessorConfig/PaymentProcessorConfig.vue:25]: Do we want :isEnabled="!isFormLoading || isFormValid"?

Comment (jsandoval81) [reply webroot/src/components/PaymentProcessorConfig/PaymentProcessorConfig.vue:25]: On other forms, the designs have called for the submit button to always be enabled and trigger the validation errors if invalid. This is just following that for consistency.

**Review by ChiefStief — APPROVED** [review]

**Comment by jsandoval81** [comment]
@jlkravitz This is ready for your review.

**Review by jlkravitz — APPROVED** [review]
No comments-looks good!

@isabeleliassen Good to merge this one.

**Review by isabeleliassen — APPROVED** [review]

**Pull request merged**